sdk/communication/azure-communication-callingserver/src/main/java/com/azure/communication/callingserver/models/AddParticipantsOptions.java

```diff
@@ -0,0 +1,120 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.communication.callingserver.models;
```

Comment:
We are removing namespace for .models / .events (as per review)
Does this also apply to Java?

Reply:
We have not observed such a requirement in the last review neither nor on the apiView comment, so we are still keeping this package.
Also, due to the nature of Maven and the SDK guideline rule, this package name is required if we want to have a folder to contain these models.